site/en/docs/extensions/mv3/manifest/manifest_version/index.md

```diff
@@ -2,27 +2,23 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest Version"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2021-04-19
 description: Reference documentation for the manifest_version property of manifest.json.
 ---
 
-One integer specifying the version of the manifest file format your package requires. As of Chrome
-18, developers _should_ specify `2` (without quotes) to use the format as described by this
-document:
+An integer specifying the version of the manifest file format your package requires. For example:
 
 ```json
-"manifest_version": 2
+"manifest_version": 3
 ```
 
-Consider manifest version 1 _deprecated_ as of Chrome 18. Version 2 is not yet _required_, but we
-will, at some point in the not-too-distant future, stop supporting packages using deprecated
-manifest versions. Extensions, applications, and themes that aren't ready to make the jump to the
-new manifest version in Chrome 18 can either explicitly specify version `1`, or leave the key off
-entirely.
+Legal values for this key are:
 
-The changes between version 1 and version 2 of the manifest file format are described in detail in
-[the `manifest_version` documentation.][1]
+* 3: Use the [Manifest V3][mv3] format and associated feature set.
```

Comment:
Can we format the `2` and `3` as code values?  (Not sure what the right syntax for this would be with our toolchain)

Reply:
Done.